recipes/halide/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "halide" %}
+{% set version = "2016.10.25" %}
+{% set sha256 = "2d4beac52b8c710e3001b61679eabe6d49bc5bf17abf9b284dbce81d8f5b762b" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/halide/Halide/archive/release_{{ version.replace('.', '_') }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - ldflags.patch
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - clangdev
+    - llvmdev
```

Comment:
clang is used to parse some things at build time, and llvm gets statically linked (effectively), so these are not runtime dependencies. There are no runtime dependecies for the static lib, and only zlib for the dylib.